src/logic/calculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
```

Comment:
## Missing Requirements 📝
![Capture](https://user-images.githubusercontent.com/93039359/207785033-31f21c38-6b73-4e2e-80cc-de05e642c993.PNG)

- [ ] You are missing the requirements shown by a red highlight in the above image 😊 Basically in this milestone you are supposed to make your calculator do all the mathematical operations 🤔 But you just imported these two files (**calculate.js & operate.js**) 😨 Kindly use them in your project to make your calculator do all the mathemtical operations 👍

**Note: If you are getting any difficulty in understanding this project or having any blocker then you can reach out to me on slack by searching my name Hammas in the full-time-students channel 😊**